src/ProjectOrigin.WalletSystem.Server/Services/REST/v1/CertificatesController.cs

```diff
@@ -164,13 +168,38 @@ public record GetCertificatesQueryParameters
     /// </summary>
     public int? Limit { get; init; }
 
+    /// <summary>
+    /// Which type to sort by
+    /// </summary>
+    public CertificateSortBy SortBy { get; init; }
+
+    /// <summary>
+    /// The order to sort by
+    /// </summary>
+    public CertificateSort Sort { get; init; }
+
     /// <summary>
     /// The number of items to skip.
     /// </summary>
     [DefaultValue(0)]
     public int Skip { get; init; }
 }
 
+[JsonConverter(typeof(StringEnumConverter))]
+public enum CertificateSortBy
+{
+    End,
+    Quantity,
+    Type
+}
+
+[JsonConverter(typeof(StringEnumConverter))]
+public enum CertificateSort
```

Comment:
Maybe make this a general enum and put it under shared models? And call it something more general like AscOrDesc

Reply:
fixed